install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -93,13 +93,62 @@ spec:
-          for: 15m
+            (
+              1
+              -
+              sum(
+                node_memory_MemFree_bytes
+                + node_memory_Buffers_bytes
+                + node_memory_Cached_bytes
+                AND on (instance)
+                label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+              ) / sum(
+                node_memory_MemTotal_bytes
+                AND on (instance)
+                label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+              )
+            ) * 100 > 60
+          for: 1h
           labels:
             severity: warning
           annotations:
-            message: "Memory usage of {{ $value | humanize }} on {{ $labels.node }} exceeds 90%. Master nodes starved of memory could result in degraded performance of the control plane."
+            summary: >-
+              Memory utilization across all control plane nodes is high, and could impact responsiveness and stability.
+            description: >-
+              Given three control plane nodes, the overall memory utilization may only be about 2/3 of all available capacity.
+              This is because if a single control plane node fails, the kube-apiserver and etcd my be slow to respond.
+              To fix this, increase the memory of the control plane nodes.
+    - name: extremely-high-control-plane-memory
+      rules:
+        - alert: ExtremelyHighControlPlaneMemory
+          expr: |
+            (
+              1
+              -
+              sum(
+                node_memory_MemFree_bytes
+                + node_memory_Buffers_bytes
+                + node_memory_Cached_bytes
+                AND on (instance)
+                label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+              ) / sum(
+                node_memory_MemTotal_bytes
+                AND on (instance)
+                label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+              )
+            ) * 100 > 90
+          for: 30m
+          labels:
+            severity: critical
+          annotations:
+            summary: >-
+              Memory utilization across all control plane nodes is extremely high, and could impact responsiveness and stability.
+            description: >-
+              Extreme memory utilisation on control plane nodes.
+              This can lead to cluster instability and slow responses from kube-apiserver or failing requests specially on etcd.
+              To fix this, increase the CPU and memory on your control plane nodes.
```

Comment:
nit: there are separate CPU alerts.  To address this memory alert, all you need to scale up is memory.  Can we drop "the CPU and" here, and above?